### PR TITLE
[AIRFLOW-4688]: Fix Pylint checks on modules under `scripts` folder

### PR DESCRIPTION
**Pull request by gdevanla**
Fix Pylint checks. To achieve this, perform minor re-arrangement of code snippets to allow for cleaner code. All checks under `scripts` folder has been addressed in this change.

---
Issue link: [AIRFLOW-4688](https://issues.apache.org/jira/browse/AIRFLOW-4688)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
I think there are two more files failing static checks. Can you please fix them @gdevanla ?

**Comment by gdevanla** [comment]
I observe locally the isort is using the global settings files during pre-commit run. 

Shouldn't airflow have its own project specific isort.cfg?

File: scripts/perf/dags/elastic_dag.py [hunk 1]
```diff
@@ -42,10 +40,13 @@ def parse_time_delta(time_str: str):
     """
     parts = RE_TIME_DELTA.match(time_str)
 
+    # pylint: disable=E7401
```
Comment (potiuk) [inline scripts/perf/dags/elastic_dag.py]: It's better to use names of the checks. This way it's cleared what it is about.

File: scripts/perf/scheduler_dag_execution_timing.py [hunk 2]
```diff
@@ -59,12 +70,16 @@ def change_state(self, key, state):
 
                 if not self.stop_when_these_completed:
                     self.log.warning("STOPPING SCHEDULER -- all runs complete")
-                    self.scheduler_job.processor_agent._done = True
+                    self.scheduler_job.processor_agent._done = True  # pylint: disable=W0212
```
Comment (potiuk) [inline scripts/perf/scheduler_dag_execution_timing.py]: Same here and other places

**Review by potiuk — COMMENTED** [review]
Thanks @gdevanla gdevanla. That's nice contribution - just one comment - to change all the pylint rules to use names.

Answering your question about isort - yep. Feel free to make a separate PR with isort.cfg !

**Comment by potiuk** [comment]
And please rebase to latest master. It contains fix for the failing build.

**Comment by gdevanla** [comment]
Thanks for the suggestion on isort. I will take a look at it when I get a chance. Do I need to submit an corresponding ticket on JIRA for that?

**Comment by potiuk** [comment]
Thanks @gdevanla

**Pull request merged**